api/swagger-spec/oapi-v1.json

```diff
@@ -17700,6 +17700,10 @@
      "customStrategy": {
       "$ref": "v1.CustomBuildStrategy",
       "description": "CustomStrategy holds the parameters to the Custom build strategy"
+     },
+     "jenkinsPipelineStrategy": {
```

Comment:
> 1) I don't think 'start build' button has any meaning here, @bparees @jimmidyson? I tried to click on that button and the build will stay in 'New' forever ;-)

I think @jstrachan and @jimmidyson have answered this but to be sure we're all on the same page.. once everything is in place, the build's existence will cause a build to run on the jenkins service, and the build object will be updated w/ the status (running, complete, failed, etc).  so i'd say so far this is working exactly as we'd expect it to (no pod should be created for the build, per my changes) and the rest is coming.